### PR TITLE
treewide: enable working, fix fixable, disable broken tests

### PR DESCRIPTION
**Pull request by oxij**
Yet another^4 followup to #39464.

### CODE REVIEW

File: pkgs/development/libraries/pango/default.nix [hunk 1]
```diff
@@ -31,6 +31,8 @@ stdenv.mkDerivation rec {
 
   configureFlags = optional stdenv.isDarwin "--without-x";
 
+  doCheck = false; # fails 1 out of 12 tests with "Fontconfig error: Cannot load default config file"
```
Comment (7c6f434c) [inline pkgs/development/libraries/pango/default.nix]: I wonder if `makeFontsConf` would help with this… (still a mergeable improvement as-is, of course)

**Review by 7c6f434c — APPROVED** [review]
I want to wait a bit in case there is some feedback… please ping me in a few days if nobody reacts.

File: pkgs/top-level/perl-packages.nix [hunk 2]
```diff
@@ -8064,6 +8064,7 @@ let
     preConfigure = ''
       substituteInPlace Makefile.PL --replace "exit 0;" ""
     '';
+    buildInputs = [ TestPod ];
```
Comment (volth) [inline pkgs/top-level/perl-packages.nix]: Does ```perlPackages.JSON``` fail on some configs?
It passes on current ```master``` (```x86_64-linux```)

File: pkgs/development/tools/misc/texinfo/common.nix [hunk 3]
```diff
@@ -33,6 +33,8 @@ stdenv.mkDerivation rec {
     installTargets="install install-tex";
   '';
 
+  checkInputs = [ procps ];
```
Comment (matthewbauer) [inline pkgs/development/tools/misc/texinfo/common.nix]: Do you know what commands are needed from  procps?

**Comment by oxij** [comment]
@7c6f434c

Ok. Thanks! (I have ~100 patches more if this, though. =/)

@volth

These changes are for stdenv built with `config.doCheckByDefault = true` set. Which is not recommended ATM as A LOT of stuff will fail to build. In ~5-10 more PRs on the topic it should become usable by the general public. :)

@matthewbauer

> Do you know what commands are needed from procps?

No, I didn't investigate. Does it make a difference? `procps` is `unixtools` on non-linux already anyway.

**Comment by 7c6f434c** [comment]
Recording/improving test suite runnability is something I consider «non-controversial, and if it is ever controversial I do want to try influencing the outcome even with my current Nixpkgs flamewar effort budget», so please ping me if anything of that kind gets stuck. I do not want to do immediate merge because some meaningful discussion with useful clarifications usully happens.

(Argh, I want a sane default for «time to let everyone ask questions even though the change has low actual impact», my mention of «several days» was clearly too long, maybe 25 hours from submission then conclusion of existing discussions is a better formulation)

**Comment by matthewbauer** [comment]
> No, I didn't investigate. Does it make a difference? `procps` is `unixtools` on non-linux already anyway.

Yeah that's okay then. IMO it is usually more helpful to be specific with these so to list out ```ps``` or ```killall```, etc.

**Comment by oxij** [comment]
@7c6f434c

Right, like I said before, it would be nice if merge windows were automated.

But it this instance it's fine anyway, I'm not in a hurry with this patchset, conflicts it generates are very easy to resolve.

@matthewbauer

I see, I'll look into it the next time I'm gonna touch texinfo.

**Comment by oxij** [comment]
Ugh. I really feel like dumping the rest of the related changes here for the ease of testing.

Would anybody object if I dump like 100 more patches into this PR?

**Comment by oxij** [comment]
Closed in favor of #44825.

**Pull request closed without merging**